index.js

```diff
@@ -231,6 +244,12 @@ const internalCaptureWebsiteCore = async (input, options, page, browser) => {
 		await page.setUserAgent(options.userAgent);
 	}
 
+	if (options.clip) {
+		viewportOptions.deviceScaleFactor = options.clip.scaleFactor;
```

Comment:
Why are you getting `.scaleFactor` from `options.clip`?

Reply:
I'm sorry, this was just my mistake.